.eslintrc

```diff
@@ -21,6 +21,7 @@
     }
   },
   "rules": {
+    "import/no-extraneous-dependencies": ["error", {"packageDir": path.join(__dirname, './')}]
```

Comment:
Note sure why you would need this? But it's certainly not working.

Reply:
eslint was showing permanent import error for monaco-editor without this. google said i should try this. what's not working?

Reply:
well, you forgot the trailing comma - so now eslint is simply not working at all.
Which kind of explains why you don't get an error now 😄